src/components/app-demo/index.tsx

```diff
@@ -10,14 +7,8 @@ export class SmoothlyAppDemo {
 	@Prop() baseUrl: string
 	render() {
 		return (
-			<App label="Smoothly Demo">
-				<a slot="nav-start" href="display">
-					Display
-				</a>
-				<a slot="nav-start" href="https://google.com">
-					External
-				</a>
-				<smoothly-room path="">
+			<smoothly-app label="Smoothly Demo">
+				<smoothly-app-room path="/google.com" label="Links">
```

Comment:
probably rename to path as it no longer links to google.com. Maybe just have the path be "/links"